subgraph.yaml

```
@@ -92,7 +92,7 @@ dataSources:
         - event: PoolCreated(indexed address)
           handler: handlePoolCreated
   - kind: ethereum/contract
-    name: WeightedPool2TokenFactory
+    name: ds-balancer-factory-pool-to-token
```

not sure, but I think WeightedPool2TokenFactory means "a weighted pool with two tokens"

```suggestion
    name: ds-balancer-factory-two-tokens
```